README.md

```diff
@@ -344,6 +344,87 @@ $ cargo install scx_rusty
 
 and `scx_rusty` will be built and installed as `~/.cargo/bin/scx_rusty`.
 
+## Checking scx_stats
+
+- To check the scheduler statistics, use the
+
+```
+scx_SCHEDNAME --monitor SEC
+```
+
+for example 0.5 - this will print the output every half a second
+
+```
+scx_bpfland --monitor 0.5
```

Comment:
Personally would remove this part with the "for example 0.5". This just makes the README bigger with not much value

Reply:
I think such an example can stay to show exactly what it is about, but if more people think the same, I will remove it.